### PR TITLE
Theme generator: show in the website main nav

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ npm run change`

#### Description of changes

- The link will appear in the left nav under Styles
- Blurbs on the Colors and Resources pages that refer/link to the theme generator
- Added links, header tags, and styles to the theme gen page to be more consistent with the rest of the website
- _Add this to the github url to test without whitespace diffs:_

> ?w=1

#### Focus areas to test

(optional)

### CODE REVIEW

File: apps/fabric-website/src/pages/Styles/ColorsPage/ColorsPage.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export class ColorsPage extends React.Component<any, any> {
         />
         <div className={ css(pageStyles.u_maxTextWidth, styles.overview) }>
           <h2 id='Overview'>Overview</h2>
-          <p>Fabric includes 9 theme colors and 11 neutral colors. Each has helper classes for text, background, border, and hover states. When selecting colors, refer to the <a className={ styles.colorsPageLink } href={ 'https://static2.sharepointonline.com/files/fabric/fabric-website/files/coloraccessibility_29sep2016.pdf' }>color accessibility guidance (PDF)</a> to ensure that your text can be ready by everyone.</p>
+          <p>Fabric includes 9 theme colors and 11 neutral colors. Each has helper classes for text, background, border, and hover states. When selecting colors, refer to the <a className={ styles.colorsPageLink } href={ 'https://static2.sharepointonline.com/files/fabric/fabric-website/files/coloraccessibility_29sep2016.pdf' }>color accessibility guidance (PDF)</a> to ensure that your text can be ready by everyone. If you need to customize your theme, see the <a className={ styles.colorsPageLink } href={ '#/styles/themegenerator' }>Theme generator</a>.</p>
```
Comment (phkuo) [inline apps/fabric-website/src/pages/Styles/ColorsPage/ColorsPage.tsx]: Is this true anymore? I thought we removed all the helper classes.

**Comment by dzearing** [comment]
[ssr-tests 16:59:07] Starting: webpack
(undefined) ../fabric-website/src/pages/PageStyles.module.scss
Module parse failed: Unexpected character '@' (1:0)
You may need an appropriate loader to handle this file type.

File: apps/fabric-website/src/pages/Styles/ColorsPage/ColorsPage.tsx (hunk 1 above)
Comment (lynamemi) [reply apps/fabric-website/src/pages/Styles/ColorsPage/ColorsPage.tsx]: You tell me :)

Comment (Jahnp) [reply apps/fabric-website/src/pages/Styles/ColorsPage/ColorsPage.tsx]: Yes, this blurb refers to Fabric Core which definitely still has these classes.

File: apps/fabric-website/src/components/App/AppState.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ export const AppState: IAppState = {
         },
         {
           title: 'Brand icons',
-          url: '#/styles/brand-icons',
+          url: '#/styles/brandicons',
```
Comment (Jahnp) [inline apps/fabric-website/src/components/App/AppState.tsx]: Discussed in person--while it does indeed make sense for these to be consistent, doing this change will also break existing links. There may be a better way to handle a change/transition like this that won't result in a break--but we should probably look at doing that in a separate PR. That said--thanks for catching this!

**Pull request merged**